WooCommerce/Classes/Authentication/Epilogue/StorePickerViewController.swift

```diff
@@ -5,42 +5,56 @@ import WordPressUI
 import Yosemite
 
 
+typealias SelectStoreClosure = () -> Void
 
-/// Allows the user to pick which WordPress.com (OR) Jetpack-Connected-Store we should set up as the Main Store.
+/// StorePickerViewControllerDelegate: the interface with operations related to the store picker
 ///
-class StorePickerViewController: UIViewController {
+protocol StorePickerViewControllerDelegate: class {
```

Comment:
This is totally nitpick, but it seems like the Kool kids now declare class only protocols with `AnyObject`:
https://useyourloaf.com/blog/class-only-protocols-in-swift-4/

Even though it does make sense to me, I haven't found any direct link to any Apple doc suggesting doing so 🤷‍♂️

Reply:
Thanks for the heads up @ctarda — I will update!